src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,45 @@
 package core.basesyntax;
 
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.stream.Stream;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String result = getStringFromFile(fromFileName);
+        Path outputFile = Paths.get(toFileName);
+        try {
+            Files.write(outputFile, result.getBytes(StandardCharsets.UTF_8));
+        } catch (IOException ioException) {
+            throw new RuntimeException(ioException);
+        }
+    }
+
+    private String getStringFromFile(String fromFileName) {
+        Path inputFile = Paths.get(fromFileName);
+        final int[] supply = {0};
+        final int[] buy = {0};
+        final int indexOperation = 0;
```

Comment:
The variables `indexOperation` and `indexValue` are not used as indexes in an array or list, they are used to access elements in an array which is always of fixed size 2. Instead of declaring them as `final int`, you could use constants to represent these values.